tests/unit/VEmojiPicker.spec.ts

```diff
@@ -1,33 +0,0 @@
-import 'mocha';
-import { expect, assert } from 'chai';
```

Comment:
The tests removed?

Reply:
sorry , it appears that this was a mistake